scripts/pylib/twister/twisterlib.py

```diff
@@ -3920,6 +3921,7 @@ class DUT(object):
     def __init__(self,
                  id=None,
                  serial=None,
+                 serial_baud=115200,
```

Comment:
I suggest using a definition for the default baud rate to avoid specifying `115200` several times in the file.